src/app/shared/actions/volume-actions/volume-resize/volume-resize.component.html

```diff
@@ -9,23 +9,18 @@ <h3 class="mat-dialog-title">
     novalidate
   >
     <div class="mat-dialog-content">
-      <div *ngIf="!volumeIsRoot && diskOfferings?.length">
-        <cs-disk-offering-selector
-          name="diskOffering"
-          [diskOfferings]="diskOfferings"
-          [ngModel]="diskOffering"
-          (change)="updateDiskOffering($event)"
-        ></cs-disk-offering-selector>
-        </div>
-      <cs-slider
-        *ngIf="isCustomizedForVolume(diskOffering) || volumeIsRoot"
+      <cs-disk-offering-selector
+        name="diskOffering"
+        [isShowSelector]="!volumeIsRoot && diskOfferings?.length"
```

Comment:
`isShowSelector` and `isShowSlider` are grammatically incorrect. Use `enable` or something similar.